net_util/src/mac.rs

```diff
@@ -10,15 +10,29 @@ use std::result::Result;
 use serde::de::{Deserialize, Deserializer, Error};
```

Comment:
Can we add `#[deny(missing_docs)]` to this source file now ?

Reply:
Having `#![deny(missing_docs)]` in `lib.rs` will deny public content without doc for all the files pertaining to that crate since `#!` is a crate level attribute. You can test by commenting the line
`/// The number of tuples (the ones separated by ":") contained in a MAC address.` from `net_util/src/mac.rs.` You should receive a compile error.

Reply:
OK, that makes sense.